tools/build/rules/android.bzl

```diff
@@ -105,3 +105,35 @@ android_native_app_glue = repository_rule(
         "ANDROID_NDK_HOME",
     ]
 )
+
+# Retrieve Vulkan validation layers from the Android NDK
+
+def _android_native_vulkan_validation_layers(ctx):
+    build = ""
+    for abi in ["armeabi-v7a", "arm64-v8a", "x86", "x86_64"]:
+
+        for layer in ["core_validation", "object_tracker", "parameter_validation", "threading", "unique_objects"]:
+            layerpath = abi + "/libVkLayer_" + layer + ".so"
+            ctx.symlink(
```

Comment:
Does this work on Windows?

Reply:
Yes, confirmed by the CI build, I manually checked that the validation layers are present in the generated APKs